prefect_bitbucket/credentials.py

```diff
@@ -35,6 +50,14 @@ class BitBucketCredentials(Block):
         default=None,
         description="Identification name unique across entire BitBucket site.",
     )
+    password: Optional[str] = Field(
```

Comment:
If a password is provided, is a token also needed? Should token be optional now also?